### PR TITLE
DOCS: Update Profile Information

### PR DESCRIPTION
**Pull request by mitrish**

### CODE REVIEW

File: content/participants/ttjc@mitrish/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Hi! I am Rishabh Mittal, let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I have solid fundamentals (HTML,CSS,JS,SQL, MongoDB, Node) and have worked with frameworks such as React, Angular, SpringBoot, working towards AWS Developer Associate. Currently Focused on learning java and improving existing skill sets.
+
+## why are you taking this challenge?
+
+I need an way where i can continue to sharpen my skills, through this challenge i can keep track of my progress.
```
Comment (khusharth) [inline content/participants/ttjc@mitrish/index.md:20]: Hey! @RishKD a small mistake here:
* I need **a** way ...

File: content/participants/ttjc@mitrish/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- [LinkedIn](https://www.linkedin.com/in/mittalrish/)
+
+### projects
+
+Starting from scratch, please observe this space for updates.
+
+
+### blogs
+
+These are some of the blogs that i have already written in the past.
```
Comment (khusharth) [inline content/participants/ttjc@mitrish/index.md:42]: You have added blogs which were done before jobChallenge. Only blogs/projects done during the challenge are accepted. Kindly remove them.

**Comment by mitrish** [comment]
I have made the suggested changes. Also, i haven't set my personal profile in the terminal, so the commit is coming from my work_id, the next ones will come from @mitrish 
Thanks

File: content/participants/ttjc@mitrish/index.md [hunk 3]
```diff
@@ -41,8 +41,5 @@ Starting from scratch, please observe this space for updates.
 
 These are some of the blogs that i have already written in the past.
```
Comment (khusharth) [inline content/participants/ttjc@mitrish/index.md:42]: You can also remove the above line.

**Pull request merged**